src/App.tsx

```diff
@@ -1,27 +1,29 @@
-import React from 'react'
-import { Route, HashRouter } from 'react-router-dom'
+import React from 'react';
+import { Route, HashRouter } from 'react-router-dom';
 
-import Header from './components/Navbar/Navbar'
-import Home from './components/Home/Home'
-import Search from './components/Search/Search'
-import Tickets from './components/Tickets/Tickets'
-import User from './components/User/User'
+import NavbarMobile from './components/Navbar/NavbarMobile';
+import Home from './components/Home/Home';
+import Search from './components/Search/Search';
+import Tickets from './components/Tickets/Tickets';
+import User from './components/User/User';
 
-import './styles/reset.scss'
-import './styles/styles.scss'
+import './styles/reset.scss';
+import './styles/styles.scss';
+import NavbarDesktop from './components/Navbar/NavbarDesktop';
 
 function App() {
   return (
     <>
       <HashRouter>
-        <Header />
+        <NavbarDesktop />
         <Route exact path="/" component={Home} />
         <Route path="/search" component={Search} />
         <Route path="/tickets" component={Tickets} />
         <Route path="/user" component={User} />
+        <NavbarMobile />
```

Comment:
Possibly it should be below NavbarDesktop? 

Or create a component <Navbar /> and put there two theese components - I think that this way should look clearly

Also possibly we should use {children} like this:
<HashRouter>
        {children}
        <Route exact path="/" component={Home} />
        <Route path="/search" component={Search} />
        <Route path="/tickets" component={Tickets} />
        <Route path="/user" component={User} />
....

- and get children from the props?